data/emea-on-rails/emea-on-rails-2021/videos.yml

```
@@ -0,0 +1,143 @@
+---
+- title: "Democratizing the Fight Against Ruby Memory Bloat"
+  raw_title: Democratizing the Fight Against Ruby Memory Bloat - Hongli Lai
+  speakers:
+    - Hongli Lai
+  event_name: EMEA on Rails 2021
+  published_at: "2021-10-19"
```

I know this key name is not ideal (we should look at #134 again) but it is used as the `date` on which the talk was given, would you mind updating all the `published_at` to `"2021-06-09"`? Thank you!

```suggestion
  published_at: "2021-06-09"
```